api/comments.rb

```diff
@@ -62,5 +62,4 @@
       pass
     end
   end
-  comment.to_hash.to_json
```

Comment:
I'm going to restore this and see if this fixes the issue we saw on the sandbox.